api/src/migrations/api/1611329857558-initialDataMetadataEntities.ts

```diff
@@ -0,0 +1,89 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class initialDataMetadataEntities1611329857558 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`
+      CREATE TYPE "status" AS ENUM (
+        'created',
+        'running',
+        'done',
+        'failure'
+      );
+
+      CREATE TYPE "tags" AS ENUM (
+        'bioregional',
+        'species'
+      );
+
+      CREATE TABLE "features" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "feature_data_id" uuid NOT NULL,
+        "feature_class_name" varchar,
+        "alias" varchar,
+        "description" varchar,
+        "property_name" varchar,
+        "intersection" uuid[],
+        "tag" tags NOT NULL,
+        "creation_status" status NOT NULL,
+        "created_at" timestamp NOT NULL default now(),
+        "created_by" uuid NOT NULL,
+        "modify_at" timestamp NOT NULL default now()
+      );
+
+      CREATE TABLE "scenarios" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "name" varchar,
+        "project_id" varchar NOT NULL REFERENCES "projects" ("id"),
+        "iso3" varchar(3) NOT NULL,
+        "extent" geometry NOT NULL,
+        "wdpa_filter" jsonb default NULL,
+        "wdpa_threshold" int CHECK (wdpa_threshold BETWEEN 0 AND 100),
+        "admin_region_id" uuid,
+        "runs" int NOT NULL,
+        "blm" float8 NOT NULL,
+        "aditional_fields" jsonb,
+        "status" status NOT NULL,
+        "parent_id" uuid REFERENCES "scenarios" ("id"),
+        "created_at" timestamp NOT NULL default now(),
+        "created_by" uuid NOT NULL,
+        "modify_at" timestamp NOT NULL default now()
+      );
+
+      CREATE TABLE "output_results" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "scenario_id" uuid REFERENCES "scenarios" ("id"),
+        "run_number" int,
+        "score" float8,
+        "cost" float8,
+        "planing_units" float8,
+        "connectivity" float8,
+        "connectivity_total" float8,
+        "mpm" float8,
+        "penalty" float8,
+        "shortfall" float8,
+        "missing_values" float8,
+        "aditional_fields" jsonb
+      );
+
+
+      COMMENT ON TABLE "features" IS 'Feature management';
+      COMMENT ON COLUMN "features"."property_name" IS 'properties.<column> used for setting conservation Feature type';
+      COMMENT ON TABLE "output_results" IS 'output_results metadata from marxan runs under an scenario';
+      `)
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`
+      DROP TABLE IF EXISTS "features";
+      DROP TABLE IF EXISTS "scenarios";
+      DROP TABLE IF EXISTS "output_results";
+
+
+      DROP TYPE IF EXISTS tags;
+      DROP TYPE IF EXISTS status;
```

Comment:
would you be ok to avoid the `IF EXISTS` here? these are all introduced in this migration (I will update mine to happen later than this in terms of timestamp so we can avoid having to check this), so it should be ok to insist that a plain `DROP` should work.